DI-CLI-Documentation.md

```diff
@@ -34,6 +34,9 @@ The DataIngestion CLI is packaged using GraalVM, and no additional dependencies
 
 1. Download the `nbs-di-cli` application from source installation folder.
 2. Run the below commands directly from where you downloaded the `nbs-di-cli` application.
+3. Set up system environment variable
```

Comment:
what if they're using `bashrc` files and linux machine?

Reply:
I have tested with bashrc, but it should work